tests/unit/package_managers/python/pip/test_main.py

```
@@ -985,3 +990,102 @@ def test_invalid_env_url_rejected(
+class TestDownloadDependenciesMarkerSkip:
+    """_download_dependencies skips marker-excluded requirements under binary filters.
+
+    Regression guard for the wiring in the download loop, complementing the pure
+    helper tests in test_markers.py.
+    https://github.com/hermetoproject/hermeto/issues/1570
+    """
+
+    DIGEST = "sha256:" + "0" * 64
+
+    @pytest.mark.parametrize(
+        "arch, bcrypt_expected",
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)